SIS_import/docker-compose.yml

```diff
@@ -13,4 +13,4 @@ services:
     
```

Comment:
Optional: The docker compose latest version seems to be 3.8. currently it is given 3 which will be defaults to 3.0

https://docs.docker.com/compose/compose-file/compose-versioning/#version-3

Reply:
Thanks! The docker version has been upgraded to v3.8